Sources/SwiftCompilerPluginMessageHandling/Macros.swift

```diff
@@ -167,6 +167,7 @@ extension PluginProviderMessageHandler {
         switch macroDefinition.formatMode {
         case .auto: collapseIndentationWidth = nil
         case .disabled: collapseIndentationWidth = []
+        @unknown default: fatalError()
```

Comment:
```suggestion
        #if RESILIENT_LIBRARIES
        @unknown default: fatalError()
        #endif
```

Same for the other change.

Reply:
Ah right, I forgot I added that. FWIW, the Swift 6 compiler will no longer warning about this code when building non-resiliently (the warning is suppressed now when the default is `@unknown`) so depending on how we expect people to be developing swift-syntax we could potentially drop the verbosity of the `RESILIENT_LIBRARIES` guards.

Reply:
We still support building swift-syntax with 5.8.

Reply:
I would like to keep `RESILIENT_LIBRARIES` for now so that swift-syntax can build without warnings with a Swift 5 compiler.